api/src/test/java/dk/dbc/search/work/presentation/api/jpa/JpaBase.java

```diff
@@ -142,7 +142,7 @@ public WithEnv withConfigEnv(String... envs) {
         return new WithEnv(env);
     }
 
-    public abstract BF createBeanFactory(Map<String, String> env, EntityManager em);
+    public abstract BF createBeanFactory(Map<String, String> env, EntityManager em, EntityManagerFactory emf);
```

Comment:
You should update the JavaDoc in the beginning of this file when altering the signature of this.